rules/game_process.tex

```diff
@@ -31,14 +31,20 @@ \subsection{Robot States}
   \item[Unstiff.] It helps to facilitate a consistent and safe handling of the robots for remote competition.
     During any state, if all head buttons are pressed at least one second, the robot should move to a safe seated/crouched position and unstiffen all joints.
     So while in the \texttt{unstiff} state the robot is not allowed to move in any fashion! After booting, the robots are in their \texttt{unstiff} state.
-    Pressing the chest button once while in the \texttt{unstiff} state, permits the robot to stiffen its joints and return to the \texttt{initial} state, or a state as indicated by GameController.
+    Pressing the chest button once while in the \texttt{unstiff} state, permits the robot to stiffen its joints and return to the \texttt{setup} state, or a state as indicated by GameController.
 
-  \item[Initial.] The robots are not allowed to be moving in any fashion besides initially standing up.
+    \item[Setup.] The robots are free to move at teams convenience and humans are allowed to interact with the robots. 
+    This state is not limited in time and teams have access to the field.
+    The GameController will activate this state before \texttt{initial} (i.e at the beginning of a half and during a timeout).
     Shortly pressing the chest button will switch the robot to the \texttt{penalized} state.
 
+    \item[Initial.] The robots are not allowed to move in any fashion besides initially standing up.
+    In \texttt{initial}, the robots are awaiting a visual indication from the referee which signals the transition to \texttt{ready}.  
+
   \item[Ready.] In this state, the robots walk to their legal positions for kick-off (\cf \cref{sec:kick-off}) or a penalty kick (\cf \cref{sec:penalty_kick}).
     They remain in this state, until the head referee decides that there is no significant progress, up to a maximum of \qty{\KickOffAutoTime}{\second} for a kick-off and \qty{\PenaltyKickSetupTime}{\second} for a penalty kick.
     The GameController can activate sub-states for kick-off and penalty kicks.
+    The robot can only enter this state by receiving a message from the GameController or detecting the visual signal or by receiving a message from teammates who did.
```

Comment:
Is this sentence really required? It's missing the whistle for the transition after a goal.

Reply:
Yes as there nothing else preventing educated guess of the signal without visual recognition. I will add the whistle to the list or see if I can do otherwise.